package.json

```diff
@@ -1,66 +1,71 @@
 {
   "name": "graphql-rate-limit",
-  "version": "2.0.1",
+  "version": "2.0.2",
   "description": "Add Rate Limiting To Your GraphQL Resolvers 💂‍♀",
-  "main": "build/main/index.js",
-  "typings": "build/main/index.d.ts",
-  "module": "build/module/index.js",
+  "main": "dist/index.js",
+  "typings": "dist/index.d.ts",
+  "module": "build/esm/index.js",
   "repository": "https://github.com/teamplanes/graphql-rate-limit",
   "license": "MIT",
   "keywords": [],
   "scripts": {
-    "build": "concurrently build:*",
-    "build:main": "tsc --preserveWatchOutput -p tsconfig.json",
-    "build:module": "tsc --preserveWatchOutput -p tsconfig.module.json",
-    "example": "concurrently \"yarn build:main -w\" \"cd example && yarn dev\"",
-    "fix": "run-s fix:*",
-    "test": "yarn build:main && nyc --silent ava",
-    "lint": "yarn eslint ./src/**/*.ts",
-    "watch": "yarn build:main && concurrently \"yarn build:main -w\" \"yarn test --watch --verbose\""
+    "prepare": "npm run build",
+    "build": "tsup ./src/index.ts --format esm,cjs --legacy-output --dts",
+    "example": "concurrently \"npm run build -w\" \"cd example && npm run dev\"",
+    "fix": "eslint ./src/**/*.ts --fix",
+    "test": "tsup ./src/index.ts && nyc --silent ava",
+    "lint": "eslint ./src/**/*.ts"
   },
+  "files": [
+    "dist"
+  ],
   "engines": {
-    "node": ">=8.9"
+    "node": ">=12.0"
```

Comment:
What is the dependency/change that requires this change? If it still runs on 10.5, would setting this to `>=10.5` make sense?

Reply:
Yes >=10.5 should be fine. I skipped 10 since I’ve seen every other org I know drop support already as it’s end of life is coming up on the 30th of April this year.

Reply:
Ah, gotcha, 12 is good with me!